apps/test/unit/lib/ui/SchoolInfoInterstitialTest.jsx

```diff
@@ -156,6 +157,43 @@ describe('SchoolInfoInterstitial', () => {
     );
   });
 
+  it('calls onChange function when country is changed from US to non-US', () => {
```

Comment:
this should probably be something like
```suggestion
  it('clears the school ID when country is changed', () => {
```